libr/core/cmd_section.c

```diff
@@ -276,17 +331,17 @@ static int cmd_section(void *data, const char *input) {
 		// remove all sections
 		if (input[1] == '*') {
 			r_io_section_init (core->io);
-		}
-		if (input[1] == '.') {
-			RIOSection *s = r_io_section_vget (core->io, core->offset);
-			if (!s) {
-				return 0;
-			}
-			// use offset
-			r_io_section_rm (core->io, s->id);
-		}
-		if (input[1]) {
-			r_io_section_rm (core->io, atoi (input + 1));
+		} else if (input[1] == '0' && input[2]=='x') {		//uses the offset
```

Comment:
this "0x" looks like a bad idea to me, better do that in a different command. we cant use flags if "0x" is necessary